topsort-api-v2.yml

```diff
@@ -466,6 +508,206 @@ components:
           description: >
             Human-readable explanation of or details about the error.
             The string for a given error may change over time; code should not parse or dispatch based on particular values for this field.
+    Placement:
+      type: object
+      required:
+        - path
+      properties:
+        path:
+          type: string
+          description: >
+            URL path of the page triggering the event.
+            For web apps, this can be obtained in JS using `window.location.pathname`.
+
+            For mobile apps, use the deep link for the current view, if available.
+            Otherwise, encode the view from which the event occurred in your app as a path-like string (e.g. `/root/categories/:categoryId`).
+          example: /categories/dairy
+        position:
+          type: integer
+          description: >
+            For components with multiple items
+            (i.e. search results, similar products, etc),
+            this should indicate the index of a given item within that list.
+          minimum: 1
+        page:
+          type: integer
+          description: >
+            For paginated pages, this should indicate which page number triggered the event.
+          minimum: 1
+        pageSize:
+          type: integer
+          description: >
+            For paginated pages this should indicate how many items are in each result page.
+          minimum: 1
+          example: 15
+        productId:
+          type: string
+          description: The ID of the product associated to this event.
```

Comment:
Here is not the id pf the product, because that's already part of the event. Instead is the id of the product in case you are in a PLP.

Reply:
Ahhh, so it's only meant for non-promoted impressions/clicks, where there is no `resolvedBidId`?

Reply:
It could also be for promoted products. Imagine you have a section called People also bought, where some slots are promoted

Reply:
I understand, I meant it would be redundant for promoted because we already have that info encoded in the resolved bid ID (maybe not very accessible for analytics though).

Reply:
Updated with the purpose of this field.